deploy/osd-26958/config.yaml

```diff
@@ -0,0 +1,11 @@
+---
+deploymentMode: "SelectorSyncSet"
+selectorSyncSet:
+  resourceApplyMode: Sync
+  matchExpressions:
+    - key: hive.openshift.io/version-major-minor
+      operator: In
+      values: ["4.11", "4.12", "4.13", "4.14", "4.15", "4.16", "4.17"]
```

Comment:
I am trying to think of a way to only reinstall this on the clusters that need it instead of the whole fleet. We have ~240 or so that need it https://promlens.devshift.net/?l=U0l3r_BuBf4. This gives us the external ID, trying to see if we can easily translate to the internal ID then we can use another `matchExpressions` here to filter it down.

Reply:
Put the IDs in app-interface please.

Reply:
Scratch that, we have ~45 clusters failing, which is even more reason to not send this across 2000 clusters. And good call on not putting them here since this is an OSS repo.